plugins/modules/inventory_intent.py

```diff
@@ -2206,6 +2406,228 @@ def get_diff_deleted(self, config):
                                 self.msg = "Device Deletion get failed."
                             self.log(self.msg)
                             break
+                    self.result['msg'] = self.msg
+
+        return self
+
+    def verify_diff_merged(self, config):
+        """
+        Verify the merged status(Addition/Updation) of Devices in Cisco DNA Center.
+        Args:
+            - self (object): An instance of a class used for interacting with Cisco DNA Center.
+            - config (dict): The configuration details to be verified.
+        Return:
+            - self (object): An instance of a class used for interacting with Cisco DNA Center.
+        Description:
+            This method checks the merged status of a configuration in Cisco DNA Center by retrieving the current state
+            (have) and desired state (want) of the configuration, logs the states, and validates whether the devices
+            addition/updation/provisioning operation succeed in Cisco DNA Center.
+        """
+
+        self.get_have(config)
+        self.log(str(self.have))
```

Comment:
Can you please add what we request? How do we differentiate we are printing have and want

Reply:
Added the proper message for want and have before logging their content.